explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+            }]
+        }
+    }
+}
+```
+
+Before going in detail an important paradigm is explained that is defined by the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), namely about the interaction affordances **properties, actions**, and **events**. 
+
+Each Thing and its data and functions offerings that is possible via the interface can be classified in those affordances. Sensor and/or parameter data are considered as **properties**. Functions like on/off, dimming, etc. are seen as **actions**. Data events, streams etc.  are considered as **events**. 
+
+In this context, the MyLampThing example above would result in declaring status a property, toggling an action, and overheating an event. 
+
+Now, by reading the above example TD,  one can obtain knowledge about the Thing named `MyLampThing`, including the following:
+
+- The Thing requires the use of HTTP Basic Authentication when 
+  accessing all interaction resources (announced by securityDefinitions and security).
+- The Thing provides one Property interaction resource with the name `status`.
+  - The property `status` is accessible via the HTTP protocol
+    with a GET method at URI `https://192.168.2.134/status`.
+  - Reading the property `status` will return a string value.
+- The Thing provides one Action interaction resource with the name `toggle`. 
+  - The action `toggle` is accessible via the HTTP protocol
+    with a POST method at URI `https://192.168.2.134/toggle`.
+- The Thing provides one Event interaction resource with the name `overheating`.
+  - The event `overheating` can be obtained at URI 
+    `https://192.168.2.134/oh` by using HTTP with a long polling subprotocol.
+    The subprotocol clarifies which of several possible mechanisms in HTTP
+    are used to provide "push" notifications.
+  - Each message pushed by the Thing is a string value.
+
+
+## What you can do with a Thing Description (TD)?
+
+Based on such information in a TD a WoT stack such as designed based on the [W3C Web of Things Scripting API](https://w3c.github.io/wot-scripting-api/) can simple interpret the TD content and abstract all the communication details. Developer just working with the **properties, actions**, and **events** paradigm and do not need to take care protocol specifics such as the HTTP method, the resource address, port number, etc.. The following video gives an inside about this aspect:
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/lt_P2BU8e3I/0.jpg)](http://www.youtube.com/watch?v=lt_P2BU8e3I "TD usage with a programming API")
+
+This example also shows that TDs can be used to easily onboard Things into any IoT ecosystems (e.g., IoT cloud services, edge systems, etc.). TDs helps to understand what the Things offers and how the offered data and functions can be bound to applications processes (e.g., creating service mash-ups, dashboards, etc.). The following example shows how TD are used in a node-red application:
```

Comment:
Remove word "easily", it's not helpful.

"ecosystems" -> "ecosystem" (use of "any" means you are selecting an example, each one of which is singular).  It might also be better to say "an" rather than "any".

"TDs helps" -> "TD helps a client" (plural subject, also verb help is clearer with an object; WHO is helped?)

"application processes"

"shows how a TD is"

Reply:
I think node-red is also written Node-RED usually.

Reply:
done